x-pack/filebeat/input/netflow/testdata/golden/r.pcap.golden.json

```diff
@@ -0,0 +1,685 @@
+{
+  "test_name": "r",
```

Comment:
What’s `r`? Just curious what generated this.

Reply:
That shouldn't be there. I was just playing around with a random pcap and the golden file was generated by mistake.